src/Icons/src/Icon.php

```diff
@@ -193,16 +193,6 @@ public function __toString(): string
         return $this->toHtml();
     }
 
-    public function serialize(): string
-    {
-        return serialize([$this->innerSvg, $this->attributes]);
-    }
-
-    public function unserialize(string $data): void
-    {
-        [$this->innerSvg, $this->attributes] = unserialize($data);
-    }
-
```

Comment:
Those methods were here for caching purposes no?

Reply:
It's my understanding that the `Serializable` interface is being _[phased out](https://wiki.php.net/rfc/phase_out_serializable)_.